### PR TITLE
Bugfix for name_to_initials()

### PR DESCRIPTION
**Pull request by rabelux**
I don't know why I thought using a list comprehension and f-strings improves the code, following the KISS principle the change I made in this commit is all that should have been done in the first place. So please dump my previous PR. Tested in Python 2.7 REPL.

### CODE REVIEW

**Comment by djdembeck** [comment]
Nice! Did you test your previous cases again with this change?

```
        Examples:
            'Arthur Conan Doyle' -> 'A.C.Doyle'
            'J.K. Rowling'       -> 'J.K.Rowling'
            'J. R. R. Tolkien'   -> 'J.R.R.Tolkien'
```

**Comment by rabelux** [comment]
Yes I did and I'd suggest to expand the examples with any edge case that gets discovered. 
Could you create a doctoring including an example for `clear_contributor_text()`?

**Comment by djdembeck** [comment]
>  Could you create a doctoring including an example for `clear_contributor_text()`?
Can you rephrase this? I'm not sure I understand.

**Comment by rabelux** [comment]
I meant docstring aka function description. Clear now?

**Comment by djdembeck** [comment]
> I meant docstring aka function description. Clear now?

Yes! Thank you for clarifying. I will add one after merging this.

Thanks again for the PR!
